integration_tests/dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'sap_source_integration_tests'
-version: '0.1.0'
+version: '0.1.1'
```

Comment:
We won't be cutting a release so these changes should be reverted back to v0.1.0